src/pages/my-buses/my-buses.component.ts

```diff
@@ -25,7 +27,8 @@ export class MyBusesComponent {
   trips;
   constructor(public navCtrl: NavController, private storage: Storage,
     private alertSvc: AlertService, private alertCtrl: AlertController,
-    private modalCtrl: ModalController, private tripSvc: FavoriteTripService) {
+    private modalCtrl: ModalController, private tripSvc: FavoriteTripService,
+    private stopSvc: FavoriteStopService, private routeSvc: FavoriteRouteService) {
```

Comment:
Your naming for the service instances could be a little confusing - for example, the app has a [RouteService](https://github.com/umts/pvta-multiplatform/blob/master/src/providers/route.service.ts) **and** [FavoriteRouteService](https://github.com/umts/pvta-multiplatform/blob/master/src/providers/favorite0route.service.ts).  

I'd alleviate this with names like `favRouteSvc` and `favStopSvc`.